CHANGELOG.md

```diff
@@ -9,6 +9,7 @@
 ### Bugs fixed
 
 * [#1988](https://github.com/bbatsov/rubocop/issues/1988): Fix bug in `Style/ParallelAssignment` when assigning from `Module::CONSTANT`. ([@rrosenblum][])
+* [#1977](https://github.com/bbatsov/rubocop/issues/1977): Fix bugs in `Rails/Date` and `Rails/TimeZone` when using Time/Date subclasses.
```

Comment:
I think subclasses is a bit confusing here - the problem is just the same names in other namespaces.